tools/layer-zero-example/contracts/hts/KeyHelper.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./HederaTokenService.sol";
+
+abstract contract KeyHelper {
```

Comment:
Same as other, reduce to only what's needed for this example and link in comment to the larger one

Reply:
Done.